js_basics_HT/task1.js

```diff
@@ -0,0 +1,11 @@
+const name = 'Barack Obama'
+const age = 62
+const isBlack = true
+const headHair = null 
+const undefinedData = 
```

Comment:
when you use `const` you should assign value while declaring the variable. You should syntax error now

Reply:
use `let` instead
```js
let undefinedData
```
Please notice that in this case you should not use `=` operator

Reply:
is it only about 'undefinedData'?